extensions/wikia/SoundCloudTag/SoundCloudTagController.class.php

```diff
@@ -38,15 +38,18 @@ public function __construct() {
 		$this->helper = new WikiaIFrameTagBuilderHelper();
```

Comment:
@nandy-andy I can see benefits of inheriting `WikiaParserTagController`, but we currently do not use any of it's features; this inheritance can be easily removed.